### ISSUE TITLE
UI doesn't scale properly to high DPI screens

### ISSUE THREAD

**Issue by sguergachi** [yyoon/Journaley#113]
The UI doesn't scale properly on screens as high of a resolution as 2560 x 1440.
![image](https://cloud.githubusercontent.com/assets/615386/10283556/115ccf86-6b4e-11e5-87c2-880da6607f60.png)

**Comment by sguergachi** [comment]
This is an issue we should figure out if there's an easy fix. I saw that .NET 4.6 has support for hi dpi Win Form UI. Not sure if there's something we're not doing to enable proper scaling.

**Comment by jeanxpierre** [comment]
I second this issue.  I have a Surface Pro and thought this would be a perfect journaling app (the screenshots on your website looks awesome - so Kudos!)  But on my screen it looks exactly like the above.  If you are going to beta any new code, I would be happy to test.  Also, have you thought about making this a Universal Windows App and saving this data to the cloud?  It would be great to use this on my phone, (eventually hololens, etc.)  -- just an idea.

**Comment by sguergachi** [comment]
We will definitely post some test builds when this is fixed.

as for universal apps, this is more to do with @yyoon having more experience with Win Forms. UWA would require lots of learning and a complete rewrite of the UI and lots of internals, so maybe not something an open source project would tackle :wink: Though I do plan on making some UI changes to conform a bit better with the new Win 10 UI.

**Comment by jeanxpierre** [comment]
If you created a UWA and published it to the Windows store, I would buy it (as long as you priced it appropriately).  ...just saying... :)

**Comment by sguergachi** [comment]
@jeanxpierre We plan on implementing a way of donating to support development of the app, so you can decide how much the app is worth to you :)

**Comment by jeanxpierre** [comment]
Great!  If this involves a lot of coding effort, possibly crowdfund to get the fire burning hotter? (Kickstarter, indiegogo, gofundme...)

I really like Journaley's simplicity.  I hope to use it (without the UI bug) as soon as feasibly possible. :-)

**Comment by crxtrdude** [comment]
Donating is a way better option than selling this into a commercial product. Aside that this project is open source, there is such a commercial app that does all this already, it's Day One.

**Comment by jduerr** [comment]
Found Journaley today. (Awesome guys!!!)
I'm using the Surface Pro 4 and too am suffering from the impropper scaling. So have you guys some News regarding that issue?

**Comment by sguergachi** [comment]
Not yet, we're pretty busy with life stuff. We should have more time over the holiday to work on this though.

**Comment by ljrain** [comment]
Has anyone got this to work out, it is really not usable on a SurfaceBook.  I was excited that this might be the app to use... :( Hope there is an update soon.

**Comment by yyoon** [comment]
A quick update on this issue. Setting the `Form.AutoScaleMode` property to `Dpi` seems to produce a lot better results at higher-dpi settings. It does not magically solve all the scale issues, but it's a good starting point at least.

Unfortunately, I don't own a surface tablet or a surface book, so I'm testing this by changing the dpi settings on my desktop. Hopefully the app appears the same on actual surface devices.

**Comment by Faterson** [comment]
I'm also unable to use version 2.1 on Windows 10 (via Boot Camp) on the MacBook 12 Retina that has the resolution of 2304 x 1440 pixels. Everything is fine with old Journaley version 1.4 on my main desktop machine, also running Windows 10 with the resolution of 2560 x 1600. I'll try uninstalling version 2.1 from the MacBook, and installing the old version 1.4 instead.

**Comment by crxtrdude** [comment]
I would prefer though that you guys would be helping testing this out once test builds would be built.

**Comment by sguergachi** [comment]
I have a surface I can test on @yyoon

**Comment by mansart-git** [comment]
I am unable to use Journaley on my brand new Dell XPS 13 running Windows 10 at a resolution of 3200x1800. It doesn't look like the OPs screenshot. It just looks impossibly small, rendering it unusable. I'm guessing that falls under the same category. Hope it can be resolved soon and would be happy to test a beta.

**Comment by jduerr** [comment]
I have a Surface, too. looking Forward to test a new build.

**Comment by Faterson** [comment]
Absolutely, once a new build is ready, I'll be happy to test the resolution out on the MacBook. (As to running a test build regularly, I probably wouldn't dare to, as I have thousands of entries in Day One. I'm stuck using Day One from the summer of 2014 or so, because their later regular releases on iOS were so buggy I had to stop updating Day One.)

**Comment by lucasmarshall** [comment]
Just got a surface pro 4 and running into the same issue. Looking forward to testing a fix.

**Comment by eruditewriter** [comment]
Just discovered this program and was having this same scaling issue on a new Thinkpad. Resolved by going into the settings and changing the custom scale setting of text from the default 150% to 100% and it now works great (though a bit small). Easy workaround.

**Comment by Faterson** [comment]
Your workaround accomplishes nothing for me, and I trust you're not talking about Windows settings. Changing Windows settings for the sake of a single app like Journaley is not a workable solution, even if it resolved the scaling issues. Changing font sizes directly in Journaley makes no difference.

**Comment by jackfierce** [comment]
Any solutions to this problem? I am running into the same issue on a Dell XPS 9550 with a 4k screen. Let me know as I would love to use your application!
